specification/monitor/data-plane/Microsoft.Insights/preview/2023-11-01-preview/metricBatch.json

```diff
@@ -0,0 +1,562 @@
+        },
+        "average": {
+          "type": "number",
+          "format": "double",
+          "description": "The average value in the time range."
+        },
+        "minimum": {
+          "type": "number",
+          "format": "double",
+          "description": "The least value in the time range."
+        },
+        "maximum": {
+          "type": "number",
+          "format": "double",
+          "description": "The greatest value in the time range."
+        },
+        "total": {
+          "type": "number",
+          "format": "double",
+          "description": "The sum of all of the values in the time range."
+        },
+        "count": {
+          "type": "number",
+          "format": "double",
+          "description": "The number of samples in the time range. Can be used to determine the number of values that contributed to the average value."
+        }
+      },
+      "description": "Represents a metric value."
+    },
+    "QueryBatchMetadataValue": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/QueryBatchLocalizableString",
+          "description": "The name of the metadata."
+        },
+        "value": {
+          "type": "string",
+          "description": "The value of the metadata."
+        }
+      },
+      "description": "Represents a metric metadata value."
+    },
+    "QueryBatchMetric": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The metric Id."
+        },
+        "name": {
+          "$ref": "#/definitions/QueryBatchLocalizableString",
+          "description": "The name and the display name of the metric, i.e. it is localizable string."
+        },
+        "displayDescription": {
+          "type": "string",
+          "description": "Description of this metric"
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type of the metric resource."
+        },
+        "unit": {
+          "$ref": "#/definitions/QueryBatchMetricUnit",
+          "description": "The unit of the metric."
+        },
+        "timeseries": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QueryBatchTimeSeriesElement"
+          },
+          "description": "The time series returned when a data query is performed."
+        },
+        "errorCode": {
+          "type": "string",
+          "description": "'Success' or the error details on query failures for this metric."
+        },
+        "errorMessage": {
+          "type": "string",
+          "description": "Error message encountered querying this specific metric."
+        }
+      },
+      "required": [
+        "id",
+        "name",
+        "displayDescription",
+        "type",
+        "unit",
+        "timeseries"
+      ],
+      "description": "The result data of a query."
+    },
+    "QueryBatchTimeSeriesElement": {
```

Comment:
This is a duplicate of the pre-existing type TimeSeriesElement except "values" is now called "data" if we revert to "values" we can remove this duplicate